101/check-setup.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+cd $HOME
+
+tgt=$(vespa config get target -c never 2>&1)
+if [ "$tgt" != "target = cloud" ]; then
+	msg="Configure the CLI: ${tgt}"
```

Comment:
This should tell the user to configure target cloud, no? I think it will show `target = local` if it's configured like that. Maybe we can just spell out the command they should run and why?

Like:
`Configure the CLI to target Vespa Cloud: vespa config set target cloud`